src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/DefendantResponseReceiptContentProvider.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.cmc.claimstore.documents.content;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.claimstore.config.properties.notifications.NotificationsProperties;
+import uk.gov.hmcts.cmc.claimstore.services.staff.content.ClaimContentProvider;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.ResponseData;
+import uk.gov.hmcts.cmc.domain.models.legalrep.StatementOfTruth;
+
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import static java.util.Objects.requireNonNull;
+import static uk.gov.hmcts.cmc.claimstore.utils.Formatting.formatDate;
+
+@Component
+public class DefendantResponseReceiptContentProvider {
```

Comment:
I recommend merging `DefendantResponseReceiptContentProvider` with `DefendantResponseCopyContentProvider`. IMO there is no point in having two classes doing almost the same thing. The same way for me there is not point having some common / abstract content provided.

A `DefendantResponseContentProvider` should just prepare JSON object representing whole response and be reused by any PDFs focused on defendant response. Even if some fields are not consumed by particular PDF it does not matter.